linux/setup_repo.sh

```diff
@@ -3,8 +3,40 @@
 # This script installs a Jamulus repository to Debian based systems
 
 if [[ ${EUID} -ne 0 ]]; then
-     echo "Error: This script must be run as root."
-     exit 1
+    echo "Error: This script must be run as root."
+    exit 1
+fi
+
+# Check for apt version >= 2.2.0 (if found, assuming Debian-based compatible with repo)
+# Issue: https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1950095
+
+if APT_VERSION="$(apt --version)"; then
+    APT_MAJOR=$(echo "$APT_VERSION" | cut -d' ' -f 2 | cut -d'.' -f 1)
+    APT_MINOR=$(echo "$APT_VERSION" | cut -d' ' -f 2 | cut -d'.' -f 2)
+else
+    echo "This script is only compatible with Debian based distributions which have apt, but apt is not available. Please check that your OS is supported."
+    exit 1
+fi
+
+if [[ ${APT_MAJOR} -lt 2 || ( ${APT_MAJOR} -eq 2 && ${APT_MINOR} -lt 2 ) ]]; then
```

Comment:
```suggestion
if [[ ${APT_MAJOR} -lt 2 || (${APT_MAJOR} -eq 2 && ${APT_MINOR} -lt 2)   ]]; then
```
What the style checker wants